src/librustdoc/clean/mod.rs

```diff
@@ -2053,8 +2053,21 @@ pub(crate) fn clean_middle_ty<'tcx>(
         }
 
         ty::Alias(ty::Weak, data) => {
-            let ty = cx.tcx.type_of(data.def_id).subst(cx.tcx, data.substs);
-            clean_middle_ty(bound_ty.rebind(ty), cx, None, None)
+            if !cx.tcx.features().lazy_type_alias {
```

Comment:
Swap the condition and the if/else branches

Reply:
Done!